projects/m1/012-what-color-is-that-square/js/index.js

```diff
@@ -0,0 +1,42 @@
+const prompt = promptSync();
+const positionChoise = prompt('Inserisci le coordinate che vuoi trovare. Scrivi il valore delle colonne in lettere e il valore delle righe in numeri: ').toLowerCase();
+
+let xUtent = positionChoise.charAt(0);
+let yUtent = parseInt(positionChoise.charAt(1));
+
+positionX = false;
+positionY = false;
+positionYNumber = 0;
+positionXNumber = 0;
```

Comment:
It's **not a good idea** not to declare variables because they are considered **globally scoped** _(you could inadvertently change them in complex programs)_.
Please always use `let` or `const` _(exceptionally var)_.